src/main/kotlin/no/nav/familie/ef/sak/service/steg/BeslutteVedtakSteg.kt

```diff
@@ -34,22 +41,30 @@ class BeslutteVedtakSteg(private val taskRepository: TaskRepository,
     }
 
     override fun utførOgReturnerNesteSteg(behandling: Behandling, data: BeslutteVedtakDto): StegType {
-        val saksbehandler = totrinnskontrollService.lagreTotrinnskontrollOgReturnerBehandler(behandling, data)
+        val beslutter = totrinnskontrollService.lagreTotrinnskontrollOgReturnerBehandler(behandling, data)
 
         ferdigstillOppgave(behandling)
 
         return if (data.godkjent) {
             if (behandling.type != BehandlingType.BLANKETT) {
-                vedtaksbrevService.lagreEndeligBrev(behandling.id)
-                opprettTaskForIverksettMotOppdrag(behandling)
+                val fil = vedtaksbrevService.lagreEndeligBrev(behandling.id).pdf
+                require(fil != null) { "For å iverksette må det finnes en pdf" }
+                if (featureToggleService.isEnabled("familie.ef.sak.brukEFIverksett")) {
```

Comment:
Tenker vi ikke trenger featuretoggle her, dette er likevel ikke i bruk i prod?

Reply:
mest for å synke mellom oss og iverksett